### PR TITLE
Tests for web/aerocom_evaluation

### PR DESCRIPTION
**Pull request by ejgal**
Working on tests for aerocom_evaluation for #133

### CODE REVIEW

File: pyaerocom/web/test/test_aerocom_evaluation.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    from pandas import DataFrame
+    assert isinstance(table, DataFrame)
+    assert len(table) > 0
+
+def test_aerocom_evaluation_get_custom_read_method_model_file(stp, model_config_aux):
+    methods_file = '/home/eirikg/Desktop/pyaerocom/data_new/eval_py/cube_read_methods.py'
+    stp.add_methods_file = methods_file
+    stp.model_config = model_config_aux
+    fun = stp.get_custom_read_method_model('add_cubes')
+    assert fun == add_cubes
```
Comment (ejgal) [inline pyaerocom/web/test/test_aerocom_evaluation.py:82]: @jgliss: Do you have a suggestion on how to deal with needing this file for testing? Here I just copied it from the data_new repo..
Should we add it to the testdata? Create it "on the fly"?

**Comment by jgliss** [comment]
@ejgal can this be reviewed and merged (no reviewer assigned)

**Comment by ejgal** [comment]
Oh, yes it's ready for review. @jgliss

**Review by jgliss — COMMENTED** [review]
All good :)

**Pull request merged**